00-programming-fundamentals/intro-to-ruby-mine.md

```diff
@@ -0,0 +1,43 @@
+# Introduction to RubyMine
+
+RubyMine will be the application we use to read and write Ruby code from now on. It is referred to as a Ruby Integrated Development Environment (or Ruby IDE, for short). In other words, it's more than a text editor.
+We have started to get familiar with VS Code, which we will use again later on in the curriculum, but before we spend much time with Ruby we want to get comfortable with a heftier tool that understands how to read Ruby and is able to help us _debug_ Ruby programs (something we will learn how to do very soon). 
+
+VS Code is to RubyMine what a paper map is to a GPS. We _can_ build big projects in Ruby with VS Code. However, RubyMine is a tool that enables us to focus less on missing that right turn so that we can focus more on understanding the big picture of where we want to go.
+
+### Adoption at Ada
+
+Full disclosure: previous cohorts have used VS Code for the full Ruby curriculum. C14 is the first cohort to use RubyMine. We are making the switch because RubyMine will give us the ability to debug in a way we aren't able to with VS Code, which is an important skill we will learn and practice throughout the curriculum.
+
+Please note that if there are any mentions of `VS Code` or the command `code .` during the Ruby portions of the curriculum, that the document is outdated. If something in the Ruby curriculum is talking about VS Code, it is probably meant to be talking about RubyMine.
+
+## Awesome Keyboard Shortcuts
+To view all the keyboard shortcuts, navigate to `Code > Preferences > Keymap`
+Below are a few you'll likely use.
+
+Shortcut | Effect
+---      | ---
+`Double ⇧`  | Search Everywhere. Find anything related to RubyMine or your project and open it, execute it, or jump to it.
```

Comment:
Does this mean "left shift" and "right shift"?

Also, I'd recommend replacing the `⇧` symbol with the word "shift" since at least on my Air the shift key doesn't have that symbol.

Also I added [`<kbd>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd) tags for clarity.

```suggestion
<kbd>left shift</kbd> + <kbd>right shift</kbd>  | Search Everywhere. Find anything related to RubyMine or your project and open it, execute it, or jump to it.
```

Reply:
Thanks! It actually means double-click, which I can clarify.